
float64 jointpos1
float64 jointpos2
float64 jointpos3

float64 jointvel1
float64 jointvel2
float64 jointvel3

float64 jointfor1
float64 jointfor2
float64 jointfor3
